go-controller/pkg/ovn/ovn_suite_test.go

```diff
@@ -1,13 +1,120 @@
 package ovn
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/nbdb"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing/libovsdb"
+	ovntypes "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 )
 
 func TestClusterNode(t *testing.T) {
 	gomega.RegisterFailHandler(ginkgo.Fail)
 	ginkgo.RunSpecs(t, "OVN Operations Suite")
 }
+
+type option func(machine *secondaryNetworkExpectationMachine)
+
+type secondaryNetworkExpectationMachine struct {
```

Comment:
Would this fit better somewhere in `go-controller/pkg/testing`?

Reply:
I've set it here because I don't think this is to be re-used anywhere else outside this package, but, I really don't care. 

I'll put it wherever you say; just say the words.

Reply:
Ah, maybe then `ovn_test.go`